wdl-engine/src/backend/local.rs

```diff
@@ -337,4 +338,10 @@ impl TaskExecutionBackend for LocalTaskExecutionBackend {
             completed: completed_rx,
         })
     }
+
+    fn cleanup(&self, _output_dir: &Path) -> Result<oneshot::Receiver<Result<()>>> {
```

Comment:
If we take my suggestion for a default implementation of `cleanup` to return `None`, we can delete this change.